tests/driver/runner_test.py

```diff
@@ -3274,7 +3274,7 @@ async def test_raw_with_timeout_and_opaqueid(self, es):
 class SleepTests(TestCase):
     @mock.patch("elasticsearch.Elasticsearch")
     # To avoid real sleeps in unit tests
-    @mock.patch("asyncio.sleep", return_value=as_future())
+    @mock.patch("asyncio.sleep", return_value=None)
```

Comment:
Is the `return_value` parameter a leftover? In https://github.com/elastic/rally/pull/1355#issuecomment-945419821 @pquentin  mentions:

> I ended up taking advantage of the fact that @mock.patch() can handle async function just fine in Python 3.8+

So I had expected that this (and related lines) don't specify the `return_value` parameter?

Reply:
Thanks, I'm not familiar with `@mock.patch` and had not realized that. Removed them now. :+1: